spec/fixtures/files/test_api.json

```diff
@@ -131,6 +134,49 @@
         ]
       }
     },
+    "RecursiveHat": {
```

Comment:
Do tests fail for this schema without the change to `model.rb` and `_class.tmpl`?

Reply:
Tests fail without the change to model.rb, but not for _class.tmpl.